OWNERS

```diff
@@ -1,9 +1,11 @@
 reviewers:
```

Comment:
@aravindhp Based on your review on https://github.com/openshift/release/pull/3027 should @kevinrizza be added to this?

Reply:
Not needed. I wanted him on the OWNERS in openshift/release#3027 because he knows a lot about CI. But given that OWNERS file is auto-generated it is fine not including him.